src/VisualStudio/CSharp/Impl/Options/IntelliSenseOptionPageControl.xaml

```diff
@@ -60,6 +60,10 @@
                                           x:Name="Always_add_new_line_on_enter"
                                           Content="{x:Static local:IntelliSenseOptionPageStrings.Option_Always_add_new_line_on_enter}"/>
                     </StackPanel>
+
+                    <CheckBox x:Uid="Show_name_suggestions"
+                                  x:Name="Show_name_suggestions"
```

Comment:
Nit: Indent.

Reply:
I think this is indented the same as the checkboxes at the top of page...

Reply:
Unfortunately some of the other ones are formatted this way too. Oh well.

Reply:
What's the desired indentation?